shanoir-ng-front/src/app/shared/utils/images-url.util.ts

```diff
@@ -69,6 +69,6 @@ export class ImagesUrlUtil {
     public static BRUKER_ICON_PATH = 'assets/images/bruker.16x16.png';
     public static EEG_ICON_PATH = 'assets/images/icons/eeg.png';
     public static BIDS_ICON_PATH = 'assets/images/icons/bids_icon.png';
-    
```

Comment:
I try to use only icons from https://fontawesome.com/ 
Though I recognize it's sometimes difficult to find icons for those abstract concepts !

Here is what I could suggest : 
* https://fontawesome.com/icons/person-walking-arrow-right?s=solid
* https://fontawesome.com/icons/industry?s=solid
* https://fontawesome.com/icons/car-tunnel?s=solid

To use it just paste the i tag in the html ```<i class="fa-solid fa-industry"></i>```

We use the free version of this library. So if the icon doesn't show it's either that it's not free or it's because it's new and we need to update the version.